charts/zammad-ldap-sync/templates/secret.yaml

```
@@ -4,6 +4,5 @@ metadata:
   name: zammad-ldap-sync
 type: Opaque
 data:
-  spring.mail.password: {{ .Values.ldapSync.applicationYML.spring.mail.password | b64enc }}
-  spring.mail.username: {{ .Values.ldapSync.applicationYML.spring.mail.username | b64enc}}
-  zammad.token: {{ .Values.ldapSync.zammadToken | b64enc}}
\ No newline at end of file
+  spring.mail.password: {{ .Values.ldapSync.pipeline.mailPassword | b64enc }}
```

YAMLlint (1.35.1)

[error] too many spaces inside braces

(braces)